deepinv/physics/mri.py

```diff
@@ -458,11 +473,31 @@ def A_adjoint(
             x = self.from_torch_complex(FiMy)
             x = self.rss(x, multicoil=True)  # [B,1,...,H,W]
         else:
+            # Use conj as coil maps are elementwise multiplication
             SiFiMy = torch.sum(torch.conj(self.coil_maps) * FiMy, dim=1)  # [B,...,H,W]
             x = self.from_torch_complex(SiFiMy)  # [B,2,...,H,W]
 
         return self.crop(x, crop=crop)
 
+    def A_dagger(
+        self, y: Tensor, mask: Tensor = None, coil_maps: Tensor = None, **kwargs
+    ) -> Tensor:
+        r"""
+        Computes least squares solution to the MRI inverse problem, as proposed in `SENSE: Sensitivity encoding for fast MRI <https://doi.org/10.1002/(SICI)1522-2594(199911)42:5%3C952::AID-MRM16%3E3.0.CO;2-S>`_.
+
+        By default uses conjugate gradient solver. Overwrite default solver arguments by passing `kwargs`. See :func:`deepinv.optim.utils.least_squares` for details.
+
+        Optionally update MRI mask or coil sensitivity maps on the fly.
```

Comment:
```suggestion
        The MRI mask or coil sensitivity maps are updated if passed as inputs to the function.
```

I felt like this was not very clear.